lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/persistence/SQLiteWriter.java

```diff
@@ -133,22 +133,23 @@ public void clearDb() throws SQLException {
 
         sqlite.getConnection().setAutoCommit(false);
         // delete old contents
-        statement.addBatch("DROP TABLE IF EXISTS Properties;");
-        statement.addBatch("DROP TABLE IF EXISTS Node;");
-        statement.addBatch("DROP TABLE IF EXISTS Way;");
-        statement.addBatch("DROP TABLE IF EXISTS WayConsistsOf;");
-        statement.addBatch("DROP TABLE IF EXISTS Connection;");
-        statement.addBatch("DROP TABLE IF EXISTS ConnectionConsistsOf;");
-        statement.addBatch("DROP TABLE IF EXISTS ConnectionDetails;");
-        statement.addBatch("DROP TABLE IF EXISTS Restriction;");
-        statement.addBatch("DROP TABLE IF EXISTS TrafficSignals;");
-        statement.addBatch("DROP TABLE IF EXISTS Route;");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.PROPERTIES + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.NODE + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.WAY + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.WAY_CONSISTS_OF + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.CONNECTION + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.CONNECTION_CONSISTS_OF + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.CONNECTION_DETAILS + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.RESTRICTION + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.TRAFFIC_SIGNALS + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.ROUTE + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.BUILDING + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.BUILDING_CONSISTS_OF + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.ROUNDABOUT + ";");
+        statement.addBatch("DROP TABLE IF EXISTS " + TABLES.ROUNDABOUT_CONSISTS_OF + ";");
+        // TODO: Deprecated tables?
```

Comment:
Yes, old tables;. Deleting these here makes old databases consistent again after updating/importing.

Reply:
I added these to the `TABLES`-class with an `@Deprecated` annotation